src/osmium/replication/server.py

```diff
@@ -315,6 +315,8 @@ def get_state_info(self, seq=None, retries=2):
             if ts is not None and seq is not None:
                 return OsmosisState(sequence=seq, timestamp=ts)
 
+            LOG.debug("Loading state info %s retrying again (trial #%d", seq, trial)
```

Comment:
There is a closing bracket missing in log string.

Reply:
Thanks, I'm fixing it.